docs/python/settings-reference.md

```diff
@@ -19,7 +19,7 @@ For general information about working with settings in VS Code, refer to [User a
 | Setting<br/>(python.) | Default | Description |
 | --- | --- | --- |
 | condaPath | `"conda"` | Path to the `conda` executable. |
-| pythonPath | `"python"` | Path to the Python interpreter, or the path to a folder containing the Python interpreter. Can use variables like `${workspaceFolder}` and `${workspaceFolder}/.venv`. Using a path to a folder allows anyone working with a project to create an environment in the `.venv` folder as appropriate to their operating system, rather than having to specify an exact platform-dependent path. The `settings.json` file can then be included in a source code repository. |
+| defaultInterpreterPath | `"python"` | Path to the default Python interpreter to be used by the Python extension on the first time it loads for a workspace, or the path to a folder containing the Python interpreter. Can use variables like `${workspaceFolder}` and `${workspaceFolder}/.venv`. Using a path to a folder allows anyone working with a project to create an environment in the `.venv` folder as appropriate to their operating system, rather than having to specify an exact platform-dependent path. `settings.json` file can then be included in a source code repository. It's worth noting that changes to this setting made after an interpreter had been selected for a workspace will not be applied or considered by the Python extension, and that the Python extension doesn't automatically add or change this setting.|
```

Comment:
```suggestion
| defaultInterpreterPath | `"python"` | Path to the default Python interpreter to be used by the Python extension on the first time it loads for a workspace, or the path to a folder containing the Python interpreter. Can use variables like `${workspaceFolder}` and `${workspaceFolder}/.venv`. Using a path to a folder allows anyone working with a project to create an environment in the `.venv` folder as appropriate to their operating system, rather than having to specify an exact platform-dependent path. `settings.json` file can then be included in a source code repository. It's worth noting that changes to this setting made after an interpreter has been selected for a workspace will not be applied or considered by the Python extension. As well, the Python extension doesn't automatically add or change this setting.|
```